### PR TITLE
Avital/MVI-887 Logging Cleanup 

### PR DESCRIPTION
**Pull request by avitaltune**

### CODE REVIEW

**Comment by ohadtune** [comment]
Great work !!!
Some comments:
1. We're missing the additional simple format line, example:
`{"asctime": "2017-09-03 07:46:48 +0000", "levelname": "INFO", "name": "tune_mv_integration", "version": "1.21.15", "message": "Run: Starting"}
**INFO:tune_mv_integration:Run: Starting**` 
(We can ask ourself if we need it but that differ from current implementation)
Using two handlers (json and simple) https://gist.github.com/ohadtune/745ad350fb35d9e00aeafcd500024682
2. Some log lines missing `version` (running adcolony for example)
first and last lines
{"asctime": "2017-09-03 11:49:45 +0300", "levelname": "INFO", "name": "root", "version": null, "message": "cost.ironsource: Starting"}
{"asctime": "2017-09-03 11:50:14 +0300", "levelname": "INFO", "name": "tune_mv_integration.callouts.worker", "version": null, "message": "Finished", "Exit Code": 0}

Another line:
{"asctime": "2017-09-03 11:49:56 +0300", "levelname": "INFO", "name": "tune_mv_integration.support.ios_bundle_id", "version": null, "message": "IOS TrackId Lookup: Result successful", "bundle_id": "edu.self.AtomicDodgeBallLite", "track_id": 550854415}
3. both `requests-mv-integration` and `tune-mv-integration-python` need to be modified to not use removed modules
4. `config.json` can be removed
5. bump version

File: logging_mv_integrations/tune_logging.py [hunk 1]
```diff
@@ -1,49 +1,22 @@
...
 
 
 class CustomAdapter(logging.LoggerAdapter):
 
     def process(self, msg, kwargs):
-        extra = kwargs.pop('extra', self.extra['extra'])
-        version = self.extra['version']
-        return '%s, extra: %s, version: %s' % (msg, extra, version), kwargs
+        if 'extra' in kwargs:
+            kwargs['extra'].update({"version": self.extra['version']})
```
Comment (ohadtune) [inline logging_mv_integrations/tune_logging.py]: `kwargs['extra']` can be null and will fail on `update`. need to check.
maybe `if 'extra' in kwargs and kwargs['extra']`

**Comment by ohadtune** [comment]
in tune_loggin.py:
kwargs['extra'] can be null and will fail on update. need to check.
maybe if 'extra' in kwargs and kwargs['extra']

**Comment by ohadtune** [comment]
In the new implementation we're handling other libs logging. `store/google_play` for example have:
`{"asctime": "2017-09-03 13:57:09 +0300", "levelname": "INFO", "name": "oauth2client.client", "version": null, "message": "Refreshing access_token"}
`

**Comment by hasIan** [comment]
Just saw this in my email and had a small style comment related to Ohad's comment above:
```
if 'extra' in kwargs and kwargs['extra']
```
For this, I like the `get` method of dicts. You can just
```
if kwargs.get('extra')
```
or if you need the use the value
```
extra = kwargs.get('extra')
if extra:
   # Doing so many cool things with extra!
```
and it's a little more succinct. If you want a default other than `None`, you can pass it as second parameter, e.g.,
```
if _dict.get('key', False)
```

File: logging_mv_integrations/tune_logging.py [hunk 2]
```diff
@@ -1,358 +1,36 @@
...
 
-        Returns:
+    if config_path:
+        with open(config_path) as config:
+            config_dict = yaml.load(config)
 
-        """
-        # print('TuneLogging', 'create_logger')
-        logging._checkLevel(self.logger_level)
+    logging.config.dictConfig(config_dict)
```
Comment (hasIan) [inline logging_mv_integrations/tune_logging.py]: Not anything we've done, but what is up with the Python folks being internally inconsistent and introducing `camelCase` methods in `snake_casey` Python?! *Internal consistency, people, internal consistency!* (╯°□°)╯︵ ┻━┻

**Pull request closed without merging**